### PR TITLE
"Patient has already visited" feature

### PR DESCRIPTION
**Pull request by rakshakhegde**
https://www.pivotaltracker.com/story/show/164068865

### CODE REVIEW

File: app/src/test/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentSheetControllerTest.kt [hunk 1]
```diff
@@ -114,4 +114,20 @@ class RemoveAppointmentSheetControllerTest {
         MovedToPrivatePractitioner,
         Other)
   }
+
+  @Test
+  fun `when done is clicked, and a "Patient has already visited" reason is selected, then mark as already visited should be called`() {
```
Comment (vinaysshenoy) [inline app/src/test/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentSheetControllerTest.kt]: Rename test to `when done is clicked, and a "Patient has already visited" reason is selected, then appointment should be marked as visited`?

File: app/src/main/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentSheet.kt [hunk 2]
```diff
@@ -41,6 +41,7 @@ class RemoveAppointmentSheet : BottomSheetActivity() {
   @Inject
```
Comment (vinaysshenoy) [inline app/src/main/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentSheet.kt]: Reword commit to `Add "Patient already visited" to remove appointment options`?

File: app/src/test/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentSheetControllerTest.kt (hunk 1 above)
Comment (rakshakhegde) [reply app/src/test/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentSheetControllerTest.kt]: Done

File: app/src/main/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentSheet.kt (hunk 2 above)
Comment (rakshakhegde) [reply app/src/main/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentSheet.kt]: Reworded

**Pull request merged**